content/kubernetes/deployment/quick-start.md

```diff
@@ -113,21 +112,19 @@ redis-enterprise-operator   1/1     1            1           0m36s
 
 ## Create a Redis Enterprise cluster (REC)
 
-A cluster is created by creating a custom resource with the kind "RedisEnterpriseCluster"
-that contains the specification of the cluster option. See the
-[Redis Enterprise Cluster API reference](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md)
-for more information on the various options available.
+A Redis Enterprise cluster (REC) is created by with a  `RedisEnterpriseCluster` custom resource
+that contains cluster's specifications.
 
-You can test the operator by creating a minimal cluster by following this procedure:
+The following example creates a minimal Redis Enterprise cluster. See the [RedisEnterpriseCluster API reference](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md) for more information on the various options available.
 
-1. Create a file called `test-rec.yaml` that defines a Redis Enterprise cluster with three nodes:
+1. Create a file (`my-rec.yaml`) that defines a Redis Enterprise cluster with three nodes:
 
     ```sh
-    cat <<EOF > test-rec.yaml
+    cat <<EOF > <my-rec.yaml>
```

Comment:
Maybe omit the `<>` around `my-rec.yaml`? I think it's pretty obvious that this is an example filename, and it would also reduce confusion with the other `<`'s and `>`'s actually needed in this command.